linters.py

```diff
@@ -1096,6 +1096,8 @@ def _instance(self) -> Linter:
         custom_linter_module_spec = importlib.util.spec_from_file_location(
             self.__module_path.rstrip('.py').replace('/', '_'),
             self.__module_path)
+        if custom_linter_module_spec is None:
+            raise RuntimeError("Cannot find module")
```

Comment:
```suggestion
            raise RuntimeError(f'Cannot find module {self.__module_path!r}')
```

usamos comillas sencillas consistentemente. también sería buena idea imprimir qué es lo que no se encontró!